rest/taskana-rest-spring/src/main/java/pro/taskana/workbasket/rest/WorkbasketController.java

```diff
@@ -136,6 +138,27 @@
     return result;
   }
 
+  /**
+   * This endpoint deletes an existing Workbasket.
+   *
+   * <p>Returned HTTP Status codes:
+   *
+   * <ul>
+   *   <li><b>204 NO_CONTENT</b> - Workbasket has been deleted successfully
+   *   <li><b>202 ACCEPTED</b> - Workbasket still contains completed Tasks. It has been marked for
+   *       deletion and will be deleted automatically as soon as all completed Tasks are deleted.
+   *   <li><b>423 LOCKED</b> - Workbasket contains non-completed Tasks and cannot be deleted.
+   * </ul>
+   *
+   * @title Delete a Workbasket
+   * @param workbasketId the id of the Workbasket which should be deleted
+   * @return the deleted Workbasket
+   * @throws NotAuthorizedException if the current user is not authorized to delete this
+   *     Workbasket.
+   * @throws InvalidArgumentException if the requested Workbasket id is null or empty
+   * @throws WorkbasketNotFoundException if the requested Workbasket is not found
+   * @throws WorkbasketInUseException if the Workbasket contains tasks.
```

Comment:
only if there are tasks that are not yet completed, right?

Reply:
dunno. Copied from the java api documentation